common/scala/src/main/resources/application.conf

```diff
@@ -186,4 +186,14 @@ whisk {
         constraint-delimiter = " "//used to parse constraint strings
         teardown-on-exit = true //set to true to disable the mesos framework on system exit; set for false for HA deployments
     }
+
+    # tracing configuration
+    tracing {
+        cache-expiry = 30 #cache expiry in seconds. Set to appropriate value to trace long running requests
+        #zipnkin configuration. Uncomment following to enable zipkin based tracing
+        #zipkin {
+        #    url = "http://localhost:9411" //url to connecto to zipkin server
+        #    sample-rate = "0.01"
```

Comment:
Can we link to some document which explains the `sample-rate` for reference.

Reply:
I can link to javadoc for the sampler - https://github.com/openzipkin/brave/blob/master/brave/src/main/java/brave/sampler/Sampler.java#L48